### ISSUE TITLE
SSL not available for this app yet

### ISSUE THREAD

**Issue by santigraviano** [rehmatworks/serverpilot-letsencrypt#53]
Hi, I'm trying to get an SSL certificate and after the command is executed it always returns "SSL not available for this app yet."
I had tried various times and got the same result. I have others websites and I have no problem with them, only with this one.

Could you give me some clue of what this errors means? So I can try to solve it

Thanks.

**Comment by bikedorkjon** [comment]
Running into the same issue when after commenting out the code as explained in #50

**Comment by santigraviano** [comment]
So maybe I found a fix for this.

I deleted all domain related files in /etc/nginx-sp/vhosts.d which are:
domain.conf
domain.d/acme.conf
domain.d/main.conf

And I copied the same files from another domain, a fresh installed domain.
Then I replaced the domain information for the domain that I had this issue.
I ran rwsll getcert --app domain
And voila it works.

I assume that the problem was that I had old files from a prior version in /etc/nginx-sp/vhosts.d
